test/test_interfacevalues.jl

```diff
@@ -0,0 +1,166 @@
+                                        (Triangle, DiscontinuousLagrange{RefTriangle, 1}(), FaceQuadratureRule{RefTriangle}(2)),
+                                        (QuadraticTriangle, DiscontinuousLagrange{RefTriangle, 2}(), FaceQuadratureRule{RefTriangle}(2)),
+                                        (Hexahedron, DiscontinuousLagrange{RefHexahedron, 1}(), FaceQuadratureRule{RefHexahedron}(2)),
+                                        # (QuadraticQuadrilateral, Serendipity{RefQuadrilateral, 2}(), FaceQuadratureRule{RefQuadrilateral}(2)),
+                                        (Tetrahedron, DiscontinuousLagrange{RefTetrahedron, 1}(), FaceQuadratureRule{RefTetrahedron}(2)),
+                                        # (QuadraticTetrahedron, Lagrange{RefTetrahedron, 2}(), FaceQuadratureRule{RefTetrahedron}(2)),
+                                       )
+    dim = getcelltypedim(cell_shape)
+    grid = generate_grid(cell_shape, ntuple(i -> i == 1 ? 2 : 1, dim))
+    topology = ExclusiveTopology(grid)
+    for func_interpol in (scalar_interpol,#= VectorizedInterpolation(scalar_interpol)=#)
+        geom_interpol = scalar_interpol # Tests below assume this
+        n_basefunc_base = 2 * getnbasefunctions(scalar_interpol)
+        iv = Ferrite.InterfaceValues(grid, quad_rule, func_interpol, geom_interpol)
+        ndim = Ferrite.getdim(func_interpol)
+        n_basefuncs = 2 * getnbasefunctions(func_interpol)
+
+        @test getnbasefunctions(iv) == n_basefuncs
+
+        for face in topology.face_skeleton
+            neighbors = dim > 1 ? topology.face_neighbor[face[1], face[2]] : topology.vertex_neighbor[face[1], face[2]]
+            isempty(neighbors) && continue
+            other_face = neighbors[1]
+            dim == 1 && (other_face = FaceIndex(other_face[1], other_face[2]))
+            cell_coords = get_cell_coordinates(grid, face[1])
+            other_cell_coords = get_cell_coordinates(grid, other_face[1])
+            ##############
+            #   reinit!  #
+            ##############
+            reinit!(iv.face_values, cell_coords, face[2])
+            reinit!(iv.face_values_neighbor, other_cell_coords, other_face[2])
+            iv.cell_idx[] = face[1]
+            iv.cell_idx_neighbor[] = other_face[1]
+            ioi = Ferrite.InterfaceOrientationInfo(grid, face, other_face)
+            iv.ioi[] = ioi
+            ##############
+            # end reinit!#
+            ##############
+            nqp = getnquadpoints(iv)
+            # Should have same quadrature points
+            @test nqp == getnquadpoints(iv.face_values) == getnquadpoints(iv.face_values_neighbor)
+            for qp in 1:nqp
+                other_qp = Ferrite.get_neighbor_quadp(iv, qp)
+                @test getpoints(quad_rule, other_face[2])[ Ferrite.get_neighbor_quadp(iv, qp)] ≈ Ferrite.transform_interface_point(iv, getpoints(quad_rule, face[2])[qp])                
```

Comment:
If this passes I assume the transformation is working *I see it failed for 1.6, will check it. It passed locally 1.8*. How do you suggest using it? I was thinking of mutating the neighbor's `FaceValues` `qr.points` and recalculating `fv.M` when reinitializing the interface.